apcd_cms/src/apps/admin_regis_table/views.py

```diff
@@ -72,7 +72,7 @@ def get_context_data(self, registrations_content, registrations_entities, regist
         context = super(RegistrationsTable, self).get_context_data(*args, **kwargs)
 
         context['header'] = ['Business Name', 'Year', 'Type', 'Location', 'Registration Status', 'Actions']
-        context['status_options'] = ['All', 'Received', 'Processing', 'Complete']
+        context['status_options'] = ['All', 'Received', 'Processing', 'Complete', 'Withdrawn']
```

Comment:
In future, we could configure and get this list from a settings file, instead of code changes.